tss-esapi/src/error/return_code/tpm/format_zero/warning.rs

```diff
@@ -49,35 +49,38 @@ impl std::error::Error for TpmFormatZeroWarningResponseCode {}
 
 impl std::fmt::Display for TpmFormatZeroWarningResponseCode {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        // These display values are taken from the specification. Some have been
+        // changed so that the sentence starts with a capital letter and ends with
+        // period.
         match self.error_number {
-            TpmFormatZeroWarning::ContextGap => write!(f, "gap for context ID is too large"),
-            TpmFormatZeroWarning::ObjectMemory => write!(f, "out of memory for object contexts"),
-            TpmFormatZeroWarning::SessionMemory => write!(f, "out of memory for session contexts"),
-            TpmFormatZeroWarning::Memory => write!(f, "out of shared object/session memory or need space for internal operations"),
-            TpmFormatZeroWarning::SessionHandles => write!(f, "out of session handles - a session must be flushed before a new session may be created"),
-            TpmFormatZeroWarning::ObjectHandles => write!(f, "out of object handles - the handle space for objects is depleted and a reboot is required. NOTE 1: This cannot occur on the reference implementation. NOTE 2: There is no reason why an implementation would implement a design that would delete handle space. Platform specifications are encouraged to forbid it."),
```

Comment:
Yes, I think it would be a good idea to move the 'session handles' information to the Session object documentation and the 'object handles' information to the ObjectHandle type.

Reply:
fixed